TAScheduler/urlTests.py

```diff
@@ -0,0 +1,39 @@
+from django.test import TestCase
+from django.test import Client
+
+
+
+class TestUrls(TestCase):
+    def setUp(self):
+        self.client=Client()
+
+
+ #    def test_adminURL(self):
+ #
+ #       r=self.client.post('/admin/', {"username":"blah", "password":"blah"})
+ #       print(r.url)
+ #       self.assertEqual(r.url, "/admin/")
+    def test_loginTemplates(self):
+        r = self.client.get("/login/")
+        self.assertEqual(r.templates[0].name, "login.html")
+
+    def test_courseCreationTemplates(self):
+        r = self.client.get("/coursecreation/")
+        self.assertEqual(r.templates[0].name, "coursecreation.html")
```

Comment:
I think we're gonna be calling this courseManagement.html as it'll need to cover more later